_posts/2016-12-02-contributing-to-pulp-blog.md

```diff
@@ -0,0 +1,40 @@
+1) Fork [pulpproject.org respository](https://github.com/pulp/pulpproject.org/) on GitHub.
+
+2) Clone your fork to your computer.
+
+3) Change into the `pulpproject.org` directory and checkout `gh-pages` branch
+
+4) Create a new file in the `_posts` directory. The name for the file should be in the following
```

Comment:
From here down i think we want to provide information rather than steps. It could move more towards a heading format with === ====